### ISSUE TITLE
Problem with symlinks to folders

### ISSUE THREAD

**Issue by zurk** [src-d/enry#87]
### Description
If you have strange symlinks in your directory, enry can not  handle it and produce output about errors.

### Examples
Assume we are in some directory and there is no `target` subdirectory.

``` bash
mkdir temp
cd temp
ln -s ../temp/ tmp
ln -s ../target tmp1
ln -s tmp2 tmp2
cd ..
```
`ll temp` shows
```
total 24
lrwxr-xr-x  1 k  staff     8B Jul 27 11:04 tmp -> ../temp/
lrwxr-xr-x  1 k  staff     9B Jul 27 11:04 tmp1 -> ../target
lrwxr-xr-x  1 k  staff     4B Jul 27 11:04 tmp2 -> tmp2
```
And call `enry temp`  shows
```
2017/07/27 11:05:34 read /Users/k/work/rep/ast2vec/temp/tmp: is a directory
2017/07/27 11:05:34 open /Users/k/work/rep/ast2vec/temp/tmp1: no such file or directory
2017/07/27 11:05:34 open /Users/k/work/rep/ast2vec/temp/tmp2: too many levels of symbolic links
```

So, there are some problems with symlinks handling.

### So
* Can you fix it?
* Also, sometimes it is better just ignore symlinks at all. 
* May be you can add a flag for it?

**Comment by abeaumont** [comment]
Thanks for reporting @zurk. The symlink handling should be fixed (not sure it should behave differently with the `tmp2` case, since it's clearly a problem with the link itself).

I'm not so sure about whether we should handle them as normal files, ignore them or supporting both scenarios through a flag, but it's probably better to just ignore them, since they'll be duplicated files after all. @zurk would this work for you?

**Comment by zurk** [comment]
@abeaumont Yes, I am personally for ignoring.

**Comment by smola** [comment]
Ignoring is good for us too.

**Comment by abeaumont** [comment]
Fixed by #88
